docs/sources/installation/configuration.md

```diff
@@ -635,6 +635,9 @@ Secret key. e.g. AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
 ### url
 Url to where Grafana will send PUT request with images
 
+### public_url
+Url to send to users in notifications, directly appended with the resulting uploaded file name
```

Comment:
would be good to mention that this setting is optional, and by default you only need to set url